packages/perseus/src/widgets/interactive-graphs/mafs-locked-layer.tsx

```diff
@@ -0,0 +1,41 @@
+import {UnreachableCaseError} from "@khanacademy/wonder-stuff-core";
+import {Point} from "mafs";
+import * as React from "react";
+
+import type {LockedFigure} from "../../perseus-types";
+
+type Props = {
+    lockedFigures: Array<LockedFigure>;
+};
+
+const MafsLockedLayer = (props: Props) => {
+    const {lockedFigures} = props;
+    return (
+        <>
+            {lockedFigures.map((figure, index) => {
+                if (figure.type === "point") {
```

Comment:
non-blocking: You mention switch cases in the comment below, and the custom error references "case". Plus, we use switch cases elsewhere in the Mafs code for similar implementations. Why not make this a switch case?